templates/music/album_detail.html

```diff
@@ -25,54 +25,86 @@
+                   {% endfor %}
+               ],
+                    datasets: [
+                        {
+                            label: 'Streams',
+                            data: [
+                                {% for disc in album.discs %}
+                                   {% for track in disc %}
+                                       {{ track.total_streams }}{% if not forloop.last %},{% endif %}
+                                   {% endfor %}
+                                   {% if not forloop.last %},{% endif %}
+                               {% endfor %}
+                            ],
+                            backgroundColor: "#85929E"
+                        }
+                    ]
+                }
+            });
 
-        var cta = document.getElementById("myChart2").getContext('2d');
-        var myChart2 = new Chart(cta, {
-            type: 'bar',
-            data: {
-                labels: [{% for month_revenue in estimated_cumulative_revenue %}"{{ month_revenue.month }}"{% if not forloop.last %},{% endif %}{% endfor %}],
-                datasets: [
-                    {
-                        label: 'Streams',
-                        data: [{% for disc in album.discs %}{% for track in disc %}{% if track.total_streams %}{{ track.total_streams}},{% endif %}{% endfor %}{% endfor %}],
-                        backgroundColor: "#85929E"
-                    }
-                ]
-            }
-        });
-
-        var ctb = document.getElementById("myChart3").getContext('2d');
-        var myChart3 = new Chart(ctb, {
-            type: 'bar',
-            data: {
-                labels: [{% for month_revenue in estimated_cumulative_revenue %}"{{ month_revenue.month }}"{% if not forloop.last %},{% endif %}{% endfor %}],
-                datasets: [
-                    {
-                        label: 'DOWNLOADS',
-                        data: [{% for disc in album.discs %}{% for track in disc %}{% if track.total_downloads %}{{ track.total_downloads }},{% endif %}{% endfor %}{% endfor %}],
-                        backgroundColor: "#34495E"
-                    }
-                ]
-            }
-        });
+            var ctb = document.getElementById("myChart3").getContext('2d');
+            var myChart3 = new Chart(ctb, {
+                type: 'bar',
+                data: {
+                    labels: [
+                        {% for disc in album.discs %}
+                            {% for track in disc %}
+                               "{{ track.name }}"{% if not forloop.last %},{% endif %}
+                           {% endfor %}
+                           {% if not forloop.last %},{% endif %}
+                       {% endfor %}
+                    ],
+                    datasets: [
+                        {
+                            label: 'DOWNLOADS',
+                            data: [
+                                {% for disc in album.discs %}
```

Comment:
Wasn't 100% certain on this section how to do the indenting. I tried copying the formatting that you did from the previous version. But I couldn't find if you updated it since then. Let me know if there is a specific part that i'm missing on the indentation side.